src/javascript/app_2/Modules/Reports/Containers/open-positions.jsx

```diff
@@ -0,0 +1,98 @@
+import { PropTypes as MobxPropTypes }      from 'mobx-react';
+import PropTypes                           from 'prop-types';
+import React                               from 'react';
+import { withRouter }                      from 'react-router-dom';
+import { localize }                        from '_common/localize';
+import DataTable                           from 'App/Components/Elements/DataTable';
+// import { getContractPath }                 from 'App/Components/Routes/helpers';
+import { connect }                         from 'Stores/connect';
+import EmptyTradeHistoryMessage            from '../Components/empty-trade-history-message.jsx';
+import { ReportsMeta }                     from '../Components/reports-meta.jsx';
+import { getOpenPositionsColumnsTemplate } from '../Constants/data-table-constants';
+import PlaceholderComponent                from '../Components/placeholder-component.jsx';
+
+class OpenPositions extends React.Component {
+    componentDidMount()    {
+        this.props.onMount();
+    }
+
+    componentWillUnmount() { this.props.onUnmount(); }
```

Comment:
`componentDidMount` and `componentWillUnmount` is kinda inconsistent here.

Reply:
@ahkuma , this is expected due to the fact that `componentWillMount` is deprecated and we should use `componentDidMount` instead. `componentWillUnmount` is still the method to be used when removing component from `DOM`.
Refer here: https://reactjs.org/docs/react-component.html

Reply:
I was talking about the one line and multi line coding style here. So what you mean is because it is being deprecated, we make it a one liner? Woudn't a comment be better for that?

Reply:
No, my reply wasn't in regards to the structure of the function. I initially thought u were referring to the inconsistent `terminology` used in regards to `didMount` and `WillUnMount`. Now that I understand that you were referring to the structure, I think we should make it consistent and use multiline.
`` componentWillUnmount() {
        this.props.onUnmount();
    }``